README.md

```diff
@@ -299,6 +299,81 @@ remediation.proceed { (response, error) in
 }
 ```
 
+### Sign up / Register
+
+When self-service registration is enabled, the initial response will include a `.selectEnrollProfile` remediation option. Proceeding through this remediation option will return a remediation that will allow the user to supply their name and email address, allowing them to proceed through to creating a new user profile.
```

Comment:
Should we link to external public documentation on how to enable self-service registration?

Reply:
That's not a bad idea. Do you have a suggested URL? If not, I can dig one up.